### DISCUSSION TITLE
Humidity level gauge on graph

### DISCUSSION THREAD

**Discussion by Aerosoft8** [niels747/2D-Weather-Sandbox#61] [Ideas]
.

**Comment by niels747** [comment]
You want realitive humidity on the sounding graph?

**Reply by Meso22** [reply]
That would be a cool feature

**Reply by Aerosoft8** [reply]
Ofcourse. I liked the new wind gauge you added recently and  it would be nice to have a relative humidity value down there too

**Reply by niels747** [reply]
Okay, will add it
